django_netjsongraph/base/admin.py

```diff
@@ -8,6 +8,7 @@
 from django.utils.translation import ugettext_lazy as _
 from openwisp_utils.admin import ReceiveUrlAdmin
 
+from .. import settings
```

Comment:
use `from .. import settings as app_settings` for consistency with the rest of the ow codebase (settings is usually the main django settings).

Reply:
I assume this change is for all the files in the module, since other files are also doing 
```
from .. import settings
```